New mission Castling

CassYoko01 opened the pull request:
This Pull Request will:

Add the mission Castling made by CassYoko to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Gearscripts Tested
- [x] In-game Play Area Markers
- [x] In-game briefings to all players
- [x] Side-based briefings  
- [x] Mission .conf file

**Faction(s):**

BLUFOR - Task Force Omega (International Anti-Terrorist Group) - Attacking
BLUFOR Comp: Motorized Infantry Company
BLUFOR Assets: 1 STRYKER CROWS per PLT

INDFOR - OathBreakers (Modern Militia) - Defending
INDFOR Comp: Infantry Company
INDFOR Assets: 1x MMG Team and 1x Sniper Fireteam (Anti-Material Rifle)

**Objectives/Win Conditions:**

BLUFOR - Kill the HVT

INDFOR - Protect the HVT

**Terrain:**

Ringkollen

**Short mission description:**

Task Force Omega has located a key member of the Oathbreakers named Rook in northern Europe. TFO is tasked with eliminating Rook.

**Slotting Ratio:**

1:1

**Time limit:**

50 min

**Mission Briefing Screenshot:**

<img width="876" height="545" alt="Screenshot 2025-09-13 232805" src="https://github.com/user-attachments/assets/c100a48c-b6cb-4925-b0b3-85414f495d74" />

**Design concept/thoughts:**

Wanted to make a mission using a wide variety of modern weapons currently in the modpack.

**Other Notes:**

No

RandAlThor1 commented:
**QA Feedback** 

- INDFOR really isn't outgunned by BLUFOR here. adjust it either to a 3:2 or even a 2:1 for attackers. 1:1 in these woods with equal firepower. a single CROWS isn't going to fix that. Defenders in missions like these need a reason not to attempt a straight up fight against Attackers

- the MAG for the MPX in your oathbreakers gearscript is broken, probably just needs to be updated to a newer version
<img width="766" height="617" alt="image" src="https://github.com/user-attachments/assets/27607633-0ac8-428a-9738-70bc48e21ea4" />

CassYoko01 commented:
We should be all good. Please look over this again. I had to do some wacky stuff to get my GitHub working again|

Updated ratio to 2:1
Fixed mpx mag

TristGit commented:
BTW There are 60+ files, several of which are a few layers/missions that may or may not be already published or test mission files within this PR. IK youve had issues with your repo getting fudged but might want to look closer to see what youre adding.
